### DISCUSSION TITLE
Can this be used on iOS without using FCM?

### DISCUSSION THREAD

**Discussion by NGumby** [thomasgalliker/Plugin.FirebasePushNotifications#82] [General]
I'm using Azure Notification Hubs to send notifications.  Was using Plugin.PushNotifications.  Everything works fine on Android with FirebasePushNotifications.

On iOS, I send the token received in RegisteredForRemoteNotifications to my server which registers with Azure Hub.  I receive the notifications only when the app is in foreground and the notif is not displayed even if I change PresentationOptions.

Should this work or is the notification presentation done by firebase?

**Comment by thomasgalliker** [comment]
Huu, interesting question. Unfortunately, never use Azure Notification Hubs. My guess is, it should also work, no matter what product you use to send notifications. In the end, I guess, it's all up the OS native implementation to receive push notifications.

Can you give an example notification message (json) you use to send? I can try to reproduce it. (Don't forget to remove sensitive data from the notification message ;-))
It would also be helpful to know what version of the plugin you use and if you tried it on the iOS simulator or on a device.

**Comment by NGumby** [comment]
I'm using 3.0.10, but let me investigate some more before you spend time on this.  I'll let you know next week.
